pkgs/development/web/pocketbase/default.nix

```diff
@@ -0,0 +1,39 @@
+{ lib
+, buildGoModule
+, fetchFromGitHub
+}:
+buildGoModule rec {
+  pname = "pocketbase";
+  version = "0.4.2";
+
+  src = fetchFromGitHub {
+    owner = "pocketbase";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "sha256-uDseJmuK6SI3e2ICqr8SJ0iKOVCXONueZUJ6J8MKwYE=";
+  };
+
+  vendorSha256 = "sha256-8IiY/gjK8m2ntOXyG84HMiyT4GK3CgDTRG1DB+v0jAs=";
+
+  # This is the released subpackage from upstream repo
+  subPackages = [ "examples/base" ];
+
+  CGO_ENABLED = 0;
+
+  ldflags = [
+    "-s"
```

Comment:
Could you please add a comment what does these flags mean?

Reply:
Seems like all these flags are pretty common around go packages in nixpkgs? Also, they come from the upstream build instructions anyway. 

I can write the explanations if you still think that is necessary.

Reply:
Oh if there are from the upstream build instructions then you could just add a comment with a link to it :)